updated to reference write permissions (#15487)
github/docs: content/pull-requests/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/reverting-a-pull-request.md

---
## About reverting a pull request

> Reverting a pull request on {% data variables.product.product_name %} creates a new pull request that contains one revert of the merge commit from the original merged pull request. To revert pull requests, you must have [write permissions](https://docs.github.com/en/organizations/managing-access-to-your-organizations-repositories/repository-roles-for-an-organization) in the repository. 

Reviewer: No need for the domain and locale in the link:
```suggestion
Reverting a pull request on {% data variables.product.product_name %} creates a new pull request that contains one revert of the merge commit from the original merged pull request. To revert pull requests, you must have [write permissions](/organizations/managing-access-to-your-organizations-repositories/repository-roles-for-an-organization) in the repository. 
```

Author: Thank you for letting me know!

{% data reusables.repositories.sidebar-pr %}
2. In the "Pull Requests" list, click the pull request you'd like to revert.

> 3. Near the bottom of the pull request, click **Revert**. If **Revert** does not show up as an option, you will need to ask for write permissions.

Reviewer: Maybe say "displayed" rather than "show up":
```suggestion
3. Near the bottom of the pull request, click **Revert**. If the **Revert** option isn't displayed, you'll need to ask the repository administrator for write permissions.
```

Author: This suggestion makes it much more clear and sounds better. Thank you again!
